pkg/controller/datavolume-controller.go

```diff
@@ -486,6 +486,9 @@ func (c *DataVolumeController) updateDataVolumeStatus(dataVolume *cdiv1.DataVolu
 			}
 
 			_, ok := pvc.Annotations[AnnImportPod]
+			if !ok {
+				_, ok = pvc.Annotations[AnnBlankImage]
```

Comment:
This will probably need to change in the pre PR so we get the value of the annotation.